### PR TITLE
The Next Big Version of the dotfiles

### PR DESCRIPTION
**Pull request by moonglum**
* Use Ubuntu 16.04
* Use VirtualBox
* Use ansible_local
* Replace NeoVim with Vim 8
* Changed default font to Fira Mono (Nerd Font Version) and other cosmetic tweaks
* A lot of housekeeping:
  * Cleaned up the vim config **a lot** and removed almost half of the plugins
  * Cleaned up the zsh config and unified it into a single file
  * Cleaned up the other configs a little
  * Deleted files that are unused nowadays

### CODE REVIEW

File: tilde/config/nvim/init.vim [hunk 1]
```diff
@@ -26,17 +26,6 @@ colorscheme solarized
...
-" ======= Syntastic =======
-" First check JSHint, then JSCS
-let g:syntastic_javascript_checkers = ['eslint']
-
-" Also run a Syntax Check when the file is opened
-let g:syntastic_check_on_open=1
-
-" More cats
-let g:syntastic_error_symbol='🙀'
-let g:syntastic_warning_symbol='😿'
```
Comment (bitboxer) [inline tilde/config/nvim/init.vim]: The cat is gone? 😱

Comment (moonglum) [inline tilde/config/nvim/init.vim]: No worries, they'll be coming back – this is WIP 😄 I replaced `syntastic` with `neomake`. Therefore I need to configure this differently!

File: README.md [hunk 2]
```diff
@@ -59,7 +56,7 @@ I use Neovim in the terminal, because of tmux. I'm doing all my programming (cur
 
 * My terminal emulator of choice is [iTerm2](http://www.iterm2.com).
 * I use US-Extended as my keyboard layout. The German keyboard layout is useless.
-* When coding, I listen to the following music: Tron Legacy OST, Starcraft II OST, Diablo III OST, Octabitron, Saor Patrol, Schelmish, Zoë Keating, Au4, Daft Punk, LukHash, Monument Valley OST, Les Revenants OST, Apocalyptica, VNV Nation...
+* When coding, I listen to the following music: Tron Legacy OST, Starcraft II OST, Diablo III OST, Octabitron, Saor Patrol, Schelmish, ZoÃÂ« Keating, Au4, Daft Punk, LukHash, Monument Valley OST, Les Revenants OST, Apocalyptica, VNV Nation...
```
Comment (bitboxer) [inline README.md]: This is an UTF-8 problem, eh?

Comment (moonglum) [inline README.md]: Fixed, thanks!

**Pull request merged**